### ISSUE TITLE
Add spikewidgets

### ISSUE THREAD

**Issue by alejoe91** [SpikeInterface/spikely#47]
It would be great to add spikewidgets for recordings, waveforms, rasterplots, etc

**Comment by colehurwitz** [comment]
Yes, that is a good future plan once the basic GUI is more robust and useable.
